sdbuild/scripts/setup_host.sh

```diff
@@ -61,6 +59,13 @@ sudo add-apt-repository -y ppa:ubuntu-toolchain-r/ppa
 sudo apt-get update
 sudo apt-get install -y $PACKAGES
 
+# Install docker and containerd using the repository
+sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common
```

Comment:
I think one more package is missing: gnupg-agent

Reply:
I don't think `gnupg-agent` is necessary in this case. It is a daemon 
"that facilitate not having to enter the password for your key every time you want to use it." [1]
[1] https://help.ubuntu.com/community/GnuPrivacyGuardHowto#Tips_and_Tricks

Reply:
Following the official docker install recommendation.